doc/en/announce/release-8.3.3.rst

```diff
@@ -0,0 +1,21 @@
+pytest-8.3.3
+=======================================
+
+pytest 8.3.3 has just been released to PyPI.
+
+This is a bug-fix release, being a drop-in replacement. To upgrade::
+
+  pip install --upgrade pytest
+
+The full changelog is available at https://docs.pytest.org/en/stable/changelog.html.
+
+Thanks to all of the contributors to this release:
+
+* Avasam
+* Bruno Oliveira
+* Ronny Pfannschmidt
+* Sviatoslav Sydorenko (Святослав Сидоренко)
```

Comment:
It seems there's a generation issue as there's, at least, assotile too reading the changelog.

Reply:
Ahh great catch!

Seems this has changed since we introduced the backport bot... I will update the script to also consider `Co-authored-by`, which seems something sensible to do regardless of the backport or not.